CHANGELOG.md

```diff
@@ -4,6 +4,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.12.0] - 2024-04-15
+
+### Known Issues:
+
+- Default Kconfig values may not propagate correctly if the Golioth SDK
+  is not listed first in the West manifest. To workaround this, ensure
+  that Golioth is listed first in your application's manifest.
```

Comment:
This isn't an issue anymore. It's no longer relevant because client applications no longer have access to the default kconfig values used by the examples.
```suggestion
```

We should add a breaking change section describing this. Something like:
> Previously, it was possible to implicitly include default Kconfig settings from the Zephyr examples into client applications. This was unintentional and has been fixed. As a result, if your application was relying on those defaults, you'll now need to set those Kconfig values explicitly in your application.